mvp-spec/partials-updater/src/interpretor.ts

```diff
@@ -81,6 +95,34 @@ async function interpretPartialBeginToken(token: LexerToken) : Promise<string> {
     }
 }
 
+async function transformMermaids(config: PartialConfig) {
+    const lineBreak = getLineBreakSymbol(config.files[0]);
+    const cmdAndPathes = config.files.map(buildMermaidCommand);
+    cmdAndPathes.forEach((val) => {
+        child.execSync(val[0]);
+    });
+    const asyncContents = cmdAndPathes.map((val) => {
+        const assetFile = val[1];
+        const assetPath = getAssetPathRelativeToDocuments(assetFile);
+        const assetName = assetFile.substring(0, assetFile.lastIndexOf('.'));
+        const imgRef = `![${assetName}](${assetPath})`;
+        return imgRef;
+    })
+    const contents = await Promise.all(asyncContents);
+    let joined = contents.join(lineBreak);
```

Comment:
I'm concerned about this pattern of creating a variable before to return it.
Is it for debug?
Why not a simple
`return contents.join(lineBreak)`
?

Reply:
It is for debugging and sometimes semantic indeed.